core/gui/src/app/app-routing.module.ts

```diff
@@ -28,11 +28,6 @@ const routes: Routes = [
     component: WorkspaceComponent,
```

Comment:
Please fix the issue of the root URL showing a workspace without a sidebar.

![image](https://github.com/user-attachments/assets/30ea75e6-fbb0-40cf-a04f-5f7e16c1e2b9)

Reply:
With app-routing.module.ts, I can only display one component. If I choose to display the dashboard component, I cannot display anything in router-outlet and it will just show the navbar and the top bar. Displaying dashboard component with the UserWorkflowComponent isn't possible with just app-routing. If I display some other component, the dashboard component won't be displayed. After our discussion after the meeting today, do you want to display the dashboard component or the UserWorkflowComponent (without the top bar or navbar)? 

Or I can try to add some logic to display the navbar manually, but it would involve adding the navbar HTML to other components.

Reply:
You can make the root URL display the dashboard workflow section. It aligns with the issue #https://github.com/Texera/texera/issues/2566